### PR TITLE
windows cpp11

### PR DESCRIPTION
**Pull request by yhahn**
@springmeyer @BergWerkGIS consolidating all the changes needed for cpp11/vs2014 here in this branch.

### CODE REVIEW

File: scripts/vendor-node.sh [hunk 1]
```diff
@@ -6,35 +6,17 @@ if [ -z "$platform" ]; then
...
+    export INSTALL_NODE_URL=https://mapbox.s3.amazonaws.com/node-cpp11
+fi
+
 set -e -u
 set -o pipefail
 
 cwd=$(pwd)
 mkdir -p $(dirname $0)/../vendor
 cd $(dirname $0)/../vendor
+curl https://s3.amazonaws.com/mapbox/apps/install-node/v0.2.0/run |  NV=0.10.33 NP=$platform-x64 OD=$(pwd) BO=true sh
```
Comment (springmeyer) [inline scripts/vendor-node.sh]: Any reason not to start using install-node via npm (now that it is open)?

Comment (yhahn) [inline scripts/vendor-node.sh]: I prefer the curl method since it doesn't hit disk -- less to clean up later.

Comment (springmeyer) [inline scripts/vendor-node.sh]: ah, awesome.

**Comment by springmeyer** [comment]
Going to use this ticket to track overall status on visual studio 2014 builds. @yhahn and @BergWerkGIS please edit this list in place if you see other needs I'm missing.
- [x] Node v0.10.33 built against visual studio 2014 - https://github.com/mapbox/node/pull/1
- [x] Node v0.11.14 built against visual studio 2014 - https://github.com/mapbox/node/pull/2
- [x] Do we need to sign the node.exe? (currently building with `nosign`) -- signed using mapbox certs here https://github.com/mapbox/node-cpp11/blob/master/sign_node.sh
- [x] Upgrade Atom shell while we are at it? (0.19.1 is latest)
- [x] All node c++ modules built against visual studio 2014:
  - [x] node-sqlite3 built - https://github.com/mapbox/node-sqlite3/pull/363
  - [x] node-sqlite3 tag published
  - [x] node-srs built  - https://github.com/mapbox/node-srs/issues/48
  - [x] node-srs tag published
  - [x] node-mapnik built
  - [x] node-mapnik tag published
  - [x] node-gdal built - https://github.com/naturalatlas/node-gdal/pull/77 and https://github.com/naturalatlas/node-gdal/pull/76
  - [x] node-gdal tag published - https://github.com/naturalatlas/node-gdal/issues/79
- [x] Tweak studio appveyor script to install/test with visual studio 2014 node (node-cpp11)
- [x] Mapnik-omnivore appveyor tests extended to test both visual studio 2013 and 2014 binaries - https://ci.appveyor.com/project/Mapbox/mapnik-omnivore/build/1.0.175
- [x]  Fix setting of PROJ_LIB for Mapnik with visual studio 2014 - same as https://github.com/mapbox/windows-cpp-collab/issues/98
- [x] Add validation commands to publishing scripts that check for and ensure all C++ binaries dynamically link to visual studio 2014 runtimes (`dumpbin`)?
- [x] Release first test builds
- [x] Merge this branch and release new official version

### Followups
-  [x] 32 bit builds of everything.

**Comment by springmeyer** [comment]
Update here: the broken `getenv` behavior with visual studio 2014 blocked me today. Guess I need to figure this out properly: https://github.com/mapbox/windows-cpp-collab/issues/98

**Comment by springmeyer** [comment]
new build up (https://mapbox.s3.amazonaws.com/mapbox-studio/mapbox-studio-win32-x64-windows-cpp11.exe) that has the PROJ_LIB glitch but otherwise hopefully is working. Although I notice it is only 97 MB vs 147 MB (v0.1.6) which is worrysome.

**Comment by springmeyer** [comment]
> 97 MB vs 147 MB (v0.1.6) which is worrysome.

Nope, works fine :)

**Comment by yhahn** [comment]
ya 147 => 97 is @BergWerkGIS's awesome lzma switch : )

**Comment by springmeyer** [comment]
Thats right :) @BergWerkGIS FTW!

**Comment by samanpwbb** [comment]
@springmeyer ran through the builds on osx 10.10 and all seems well.

**Comment by yhahn** [comment]
- [x] tag mapnik-omnivore (on `master` currently)

**Pull request merged**